test/update.test.js

```diff
@@ -0,0 +1,67 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+require('./init.js');
+var Cloudant = require('../lib/cloudant');
```

Comment:
Capital letters first, then `_`, then lowercase letters (it's sorted correctly).

When in doubt: `:'<,'>sort`